code/modules/mob/living/carbon/xenomorph/abilities/shrieker/shrieker_powers.dm

```diff
@@ -0,0 +1,219 @@
+			if (!(XX.shriek_status_flags & SHRIEKER_SHRIEK_ENHANCED))
+				XX.armor_modifier += XENO_ARMOR_MOD_MED
+				XX.damage_modifier += XENO_DAMAGE_MOD_SMALL
+				XX.shriek_status_flags |= SHRIEKER_SHRIEK_ENHANCED
+				XX.recalculate_armor()
+				XX.recalculate_damage()
+				bufff_overlay.flick_overlay(XX, 150)
+				to_chat(XX, SPAN_XENOWARNING("You feel empowered after heearing the shriek of [src]!"))
+
+				spawn (shriek_duration)
+					XX.armor_modifier -= XENO_ARMOR_MOD_MED
+					XX.damage_modifier -= XENO_DAMAGE_MOD_SMALL
+					XX.shriek_status_flags &= ~SHRIEKER_SHRIEK_ENHANCED
+					XX.recalculate_armor()
+					XX.recalculate_damage()
+					to_chat(XX, SPAN_XENOWARNING("You feel the effects of [src] wane!"))
+			else
+				to_chat(XX, SPAN_XENOWARNING("You can only be empowered by one shriek at once!"))
+
+	else if (curr_effect_type == SHRIEKER_SHRIEK_DEBUFF)
+		playsound(xeno.loc, shriek_sound_effectt, 55, 0, status = 0)
+		xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] emits a guttural shriek!"))
+		xeno.create_shriekwave(color = "#FF0000")
+		var/slow_duration = 40
+
+		for(var/mob/living/carbon/human/human in view(3, xeno))
+			human.visible_message(SPAN_DANGER("[xeno]'s shriek shakes your entire body, causing you to fall over in pain!"))
+			if (!(xeno.shriek_status_flags & SHRIEKER_SHRIEK_DEBUFF))
+				shake_camera(human, 2, 3)
+				human.Daze(debuff_daze)
+				human.KnockDown(get_xeno_stun_duration(human, 0.5))
+				new /datum/effects/xeno_slow(human, xeno, null, null, get_xeno_stun_duration(human, slow_duration))
+
+	apply_cooldown()
+	..()
+	return
+
+
+/datum/action/xeno_action/activable/rooting_slash/use_ability(atom/A)
+	var/mob/living/carbon/Xenomorph/X = owner
+	if (!action_cooldown_check())
+		return
+
+	if (!X.check_state())
+		return
+
+	if (!check_and_use_plasma_owner())
+		return
+
+	if (!isXenoOrHuman(A) || X.can_not_harm(A))
+		to_chat(X, SPAN_XENODANGER("You must target a hostile!"))
+		return
+
+	var/mob/living/carbon/H = A
+
+	var/dist = get_dist(X, H)
+
+	if (dist > range)
+		to_chat(X, SPAN_WARNING("[H] is too far away!"))
+		return
+
+	if (dist > 1)
+		var/turf/targetTurf = get_step(X, get_dir(X, H))
+		if (targetTurf.density)
+			to_chat(X, SPAN_WARNING("You can't attack through [targetTurf]!"))
+			return
+		else
+			for (var/atom/I in targetTurf)
+				if (I.density && !I.throwpass && !istype(I, /obj/structure/barricade) && !istype(I, /mob/living))
+					to_chat(X, SPAN_WARNING("You can't attack through [I]!"))
+					return
+
+
+	if (H.stat == DEAD)
+		to_chat(X, SPAN_XENODANGER("[H] is dead, why would you want to touch it?"))
+		return
+
+	if (X.mutation_type == SHRIEKER_NORMAL)
+		var/datum/behavior_delegate/shrieker/BD = X.behavior_delegate
+		if (!istype(BD))
+			return
+
+
+
+	// Flick overlay and play sound
+	X.animation_attack_on(A, 10)
+	var/S = pick('sound/weapons/punch1.ogg','sound/weapons/punch2.ogg','sound/weapons/punch3.ogg','sound/weapons/punch4.ogg')
```

Comment:
singl letr

why is this pick even a thing? just do "punch" and it'll randomly select one of them anyways via the cursed massive if chain in playsound